pkg/google/gke/gke.go

```diff
@@ -259,3 +263,31 @@ func ListDisks(project string, zone string, service *compute.Service) ([]*comput
 	}
 	return disks, nil
 }
+
+// ListInstancesInZone will list all instances in a given zone and return a slice of MachineSpecs
+func ListInstancesInZone(projectID, zone string, c *compute.Service) ([]*MachineSpec, error) {
+	var allInstances []*MachineSpec
+	var nextPageToken string
+	log.Printf("Listing instances for project %s in zone %s", projectID, zone)
```

Comment:
I'd say either remove this logging statement or convert to `slog` to keep consistency